src/model_repository_manager/model_lifecycle.cc

```diff
@@ -92,7 +92,8 @@ VersionsToLoad(
   RETURN_IF_ERROR(GetDirectorySubdirs(model_path, &subdirs));
   std::set<int64_t, std::greater<int64_t>> existing_versions;
   for (const auto& subdir : subdirs) {
-    if (subdir == kWarmupDataFolder || subdir == kInitialStateFolder) {
+    if (subdir == kWarmupDataFolder || subdir == kInitialStateFolder ||
+        subdir == kModelConfigFolder) {
```

Comment:
Should just compile a list of folders to skip and for-loop over it.

Reply:
Can you elaborate "compile a list of folders", like create a constant vector in `src/constants.h`?

Reply:
Something like...
```
skip_folders = [kWarmupDataFolder, kInitialStateFolder, kModelConfigFolder]
if subdir in skip_folders
```

Reply:
Fixed